wp-content/themes/core/assets/css/src/theme/color/_variables.pcss

```diff
@@ -60,14 +87,22 @@
 	 * Borders
 	 * ----------------------------------------------------------------------------- */
 
-	--color-border: var(--color-grey);
+	--color-border: var(--color-grey-light);
 
 	/* -----------------------------------------------------------------------------
-	 * Media: WP Oembed
+	 * Media
 	 * ----------------------------------------------------------------------------- */
 
 	--wp-oembed-bgd: var(--color-grey);
 
+	/* -----------------------------------------------------------------------------
+	 * Alerts
+	 * ----------------------------------------------------------------------------- */
+
+	--color-alert-positive: rgba(13, 219, 145, 1);
+	--color-alert-negative: rgba(255, 61, 91, 1);
+	--color-alert-neutral: rgba(3, 134, 255, 1);
```

Comment:
Same thought, where are these used?

Reply:
These are unused at this time. They are part of design's new sink. I just added them here until we get to a place/component where they will be used. (Forms are an example, I think.)

Reply:
👍